turbo/snapshotsync/block_snapshots.go

```diff
@@ -1512,7 +1538,7 @@ func expectedTxsAmount(snapDir string, blockFrom, blockTo uint64) (firstTxID, ex
 	return
 }
 
-func TransactionsIdx(ctx context.Context, chainID uint256.Int, blockFrom, blockTo uint64, snapDir string, tmpDir string, p *background.Progress, lvl log.Lvl) (err error) {
+func TransactionsIdx(ctx context.Context, chainID uint256.Int, tx kv.Tx, blockFrom, blockTo uint64, snapDir string, tmpDir string, p *background.Progress, lvl log.Lvl, isBor bool, sprint uint64) (err error) {
```

Comment:
let's combine Bor params to 1 variable?
something like: *BorConfig
or something like: struct{ Enabled bool, Sprint uint64)